docs/docs/tutorial/intermission.md

```diff
@@ -38,11 +38,12 @@ If you haven't been through the first tutorial, or maybe you went through it on
 git clone https://github.com/redwoodjs/redwood-tutorial
 cd redwood-tutorial
 yarn install
+echo "SESSION_SECRET=$(yarn --silent rw g secret --raw)" >> .env
```

Comment:
Hmm, I don't love this complex bash command: I'm targeting the tutorial to fairly new devs and this even scares me looking at it!

I'd rather just keep the command list as it is, but add an additional paragraph afterwords explaining to run `yarn rw g secret` and to copy/paste it into your `.env` file.

Reply:
To me this command is a piece of fine-crafted art.

However i can see that it is impossible to memorize and the straighforward `yarn g secret` + copy & paste way is less intimidating and easier to memorize.

Maybe there's another place in some README or in the docs where the command could be preserved for admirers of bash kung fu? At the end of the day, such a command may come in handy for fully automated deployments and alike.

Reply:
Quick comment to add that the --silent option doesn't work on yarn 3 so it's probably best to avoid it to remain yarn version agnostic in the tutorial (not sure if we actually are yarn version agnostic though, would have to double check)

Reply:
> 

Strictly speaking the discussed commands listed in intermission are related to the "example repo". And this very repo doesn't use Yarn 3 per default, at least until now.